PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1534,6 +1534,13 @@ void Notepad_plus::doTrim(trimOp whichPart)
 	}
 }
 
+void Notepad_plus::eol2ws()
+{
+	bool isEntireDoc = _pEditView->execute(SCI_GETANCHOR) == _pEditView->execute(SCI_GETCURRENTPOS);
+	isEntireDoc ? _pEditView->execute(SCI_TARGETWHOLEDOCUMENT) : _pEditView->execute(SCI_TARGETFROMSELECTION);

```

Comment:
Could also be:
`_pEditView->execute(isEntireDoc ? SCI_TARGETWHOLEDOCUMENT : SCI_TARGETFROMSELECTION);`